src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
```

Comment:
1. add font-family here
2. it's a good practice to set for all elements `box-sizing: border-box`